src/views/Property/PropertyView.js

```diff
@@ -0,0 +1,377 @@
+import React, { useContext, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import * as axios from "axios";
+import Toast from '../../utils/toast';
+import UserContext from '../../UserContext';
+import * as Yup from "yup";
+import ToggleEditTable from "../../components/ToggleEditTable";
+import { useCalendarState } from "../../components/CalendarModal/CalendarModal";
+import PropertyManagerCard from "../../components/PropertyManagerCard/PropertyManagerCard.js";
+import BootstrapTable from 'react-bootstrap-table-next';
+import { Link } from "react-router-dom";
+import ManagerSearchPanel from '../../components/ManagerSearchPanel/ManagerSearchPanel.js';
+import RemoveTenantButton from './RemoveTenantButton.js';
+
+
+import './PropertyView.scss';
+
+
+const validationSchema = Yup.object().shape({
+  propertyName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter Property Name"),
+  propertyAddress: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Address is required"),
+  propertyUnits: Yup.string()
+    .min(
+      0,
+      "*Property must contain at least 1 unit",
+    )
+    .max(3, "*Numbers can't be longer than 3 digits")
+    .required("*Number of units is required"),
+});
+
+
+
+
+const Property = () => {
+  const userContext = useContext(UserContext);
+
+  const { id } = useParams();
+  const propertyId = id;
+
+
+  const [isEditing, setEditingStatus] = useState(false);
+  const [property, setProperty] = useState('');
+  const [tenantArray, setTenants] = useState('');
+  const calendarState = useCalendarState(property?.dateTimeStart, property?.dateTimeEnd)
+
+
+  // Error handler for axios requests
+  const axiosErrorHandler = (error) => {
+    Toast(error.message, "error");
+    return Promise.reject({ ...error });
+  };
+
+  // Handle axios errors
+  const client = axios.create();
+  client.interceptors.response.use(
+    success => success,
+    error => axiosErrorHandler(error)
+  );
+
+  /**
+   * Handle activating edit form
+   */
+  const handleEditToggle = () => setEditingStatus(!isEditing);
+
+  const onCancelClick = () => {
+    setEditingStatus(false);
+  };
+
+
+  useEffect(() => {
+    const getProperty = async () => {
+
+      const propertyResponse = await axios
+        .get(`${process.env.REACT_APP_PROXY}/api/properties/${propertyId}`,
+          { headers: { Authorization: `Bearer ${userContext.user.accessJwt}` } });
+
+      const property = propertyResponse.data;
+
+      setProperty(property)
+      return property;
+    }
+
+    const getTenants = async (property) => {
+
+      const tenantResponses = await Promise.all(property.tenantIDs.map(tenantID => axios.get(`${process.env.REACT_APP_PROXY}/api/tenants/${tenantID}`,
```

Comment:
So since we're not returning the tenantID's on this request, the page breaks

Reply:
Not sure when this changed, but the id's are present now